taskt/Core/Automation/Commands/DataTable Commands/CreateDataTableCommand.cs

```diff
@@ -1,110 +1,98 @@
 ﻿using System;
-using System.Xml.Serialization;
-using System.Data;
 using System.Collections.Generic;
-using System.Windows.Forms;
-using taskt.UI.Forms;
-using taskt.UI.CustomControls;
-using System.Drawing;
+using System.Data;
 using System.Linq;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Script;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("DataTable Commands")]
-    [Attributes.ClassAttributes.Description("This command created a DataTable with the column names provided")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to create a new DataTable")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("DataTable Commands")]
+    [Description("This command creates a DataTable with the Column Names provided.")]
+
     public class CreateDataTableCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Indicate DataTable Name")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate a unique reference name for later use")]
-        [Attributes.PropertyAttributes.SampleUsage("vMyDatatable")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        public string v_DataTableName { get; set; }
+        [PropertyDescription("New DataTable Name")]
+        [InputSpecification("Indicate a unique reference name for later use.")]
+        [SampleUsage("MyDatatable")]
```

Comment:
Variable Name can also be provided through a variable like {vDataTableName}.
Or do we need to force user to use text only not a variable in such a case?

Reply:
This is text only for now.